Patch-Source: https://gitlab.alpinelinux.org/alpine/abuild/-/merge_requests/193
--
From 7b78ac989156d1340292fd9eb6d896165c362a64 Mon Sep 17 00:00:00 2001
From: psykose <alice@ayaya.dev>
Date: Thu, 13 Apr 2023 05:04:19 +0000
Subject: [PATCH] abuild: pass --no-warnings to apk index

apk 2.14 now warns on missing deps in the same repo even with --quiet:

WARNING: No provider for the dependencies:
  /bin/sh aardvark-dns abseil-cpp-dev acl acl-dev alsa-lib-dev android-tools aom-dev apache2 at-spi2-core at-spi2-core-dev atomicparsley attr attr-dev
  audacious autoconf avahi avahi-dev aws-c-cal-dev aws-c-compression-dev aws-checksums-dev baloo-dev bash bc binutils binutils-dev black blas-dev bluez
---
 abuild.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/abuild.in b/abuild.in
index 849d938..a0026e6 100755
--- a/abuild.in
+++ b/abuild.in
@@ -1827,7 +1827,7 @@ update_abuildrepo_index() {
 		if [ -f APKINDEX.tar.gz ]; then
 			oldindex="--index APKINDEX.tar.gz"
 		fi
-		( $APK index $ABUILD_APK_INDEX_OPTS --quiet $oldindex --output APKINDEX.tar.gz.$$ \
+		( $APK index $ABUILD_APK_INDEX_OPTS --no-warnings --quiet $oldindex --output APKINDEX.tar.gz.$$ \
 			--description "$DESCRIPTION" --rewrite-arch $i *.apk && \
 		  msg "Signing the index..." && \
 		  abuild-sign -q APKINDEX.tar.gz.$$ && \
-- 
2.40.0

